### PR TITLE
fix syllabus summary actionable item cells

### PR DESCRIPTION
**Pull request by bluwave**
refs: MBL-13714
affects: Parent
release note: none

### CODE REVIEW

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Build failed, skipping coverage check
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: Parent

[MBL-13714](https://instructure.atlassian.net/browse/MBL-13714)

<details><summary>:x: XCTest failed: StudentUnitTests/SyllabusActionableItemsViewControllerTests/testRender</summary>


    Assertion Failure: StudentSyllabusViewControllerTests.swift:108: XCTAssertEqual failed: ("nil") is not equal to ("Optional("May 18, 2018 at 6:00 AM")")
    Assertion Failure: StudentSyllabusViewControllerTests.swift:109: XCTAssertEqual failed: ("nil") is not equal to ("Optional(<UIImage:0x600002bea760 named(com.instructure.icanvas.Core: calendarMonthLine) {24, 24}>)")
    Assertion Failure: StudentSyllabusViewControllerTests.swift:112: XCTAssertEqual failed: ("nil") is not equal to ("Optional("some assignment")")
    Assertion Failure: StudentSyllabusViewControllerTests.swift:113: XCTAssertEqual failed: ("nil") is not equal to ("Optional("No Due Date")")
    Assertion Failure: StudentSyllabusViewControllerTests.swift:114: XCTAssertEqual failed: ("nil") is not equal to ("Optional(<UIImage:0x600002bbfc30 named(com.instructure.icanvas.Core: assignmentLine) {24, 24}>)")
    Assertion Failure: StudentSyllabusViewControllerTests.swift:115: XCTAssertEqual failed: ("Optional(<UIDynamicSystemColor: 0x60000199e520; name = systemBlueColor>)") is not equal to ("Optional(UIExtendedSRGBColorSpace 1 0 0 1)")
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Core/Core/Syllabus/SyllabusActionableItemsCell.xib [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                <outlet property="itemNameLabel" destination="T12-iA-cYO" id="iuW-L3-EYF"/>
+                <outlet property="stackView" destination="EaW-6y-Jo2" id="UsN-fq-j6Q"/>
+            </connections>
+        </placeholder>
+        <placeholder placeholderIdentifier="IBFirstResponder" id="-2" customClass="UIResponder"/>
+        <view contentMode="scaleToFill" id="iN0-l3-epB">
+            <rect key="frame" x="0.0" y="0.0" width="414" height="100"/>
+            <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+            <subviews>
+                <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" spacing="4" translatesAutoresizingMaskIntoConstraints="NO" id="EaW-6y-Jo2">
```
Comment (armstrongnate) [inline Core/Core/Syllabus/SyllabusActionableItemsCell.xib]: I think the spacing should be 2. It is for Grades. It looks too spread out to me. Can you double check with Dave?

Comment (bluwave) [reply Core/Core/Syllabus/SyllabusActionableItemsCell.xib]: this is how he wanted it.  I put it at 2 and he thought it was too close

Comment (bluwave) [reply Core/Core/Syllabus/SyllabusActionableItemsCell.xib]: it changes daily

Comment (armstrongnate) [reply Core/Core/Syllabus/SyllabusActionableItemsCell.xib]: ah I see

**Pull request merged**